sample/src/main/kotlin/com/slack/circuit/sample/petdetail/PetPhotoCarousel.kt

```diff
@@ -186,7 +180,8 @@ internal fun PetPhotoCarousel(state: PetPhotoCarouselScreen.State) {
             val pageOffset = calculateCurrentOffsetForPage(page).absoluteValue
 
             // We animate the scaleX + scaleY, between 85% and 100%
-            lerp(start = 0.85f, stop = 1f, fraction = 1f - pageOffset.coerceIn(0f, 1f)).also { scale
+            lerp(start = 0.85f, stop = 1f, fraction = 1f - pageOffset.coerceIn(0f, 1f)).also {
+              scale,
               ->
```

Comment:
Nit: Not a fan of this spacing, maybe something more like:
```suggestion
            lerp(
              start = 0.85f,
              stop = 1f,
              fraction = 1f - pageOffset.coerceIn(0f, 1f)
            ).also { scale ->
```
But I don't think spotless would be happy with my suggestion, so this is fine.